README.md

```diff
@@ -4,24 +4,29 @@ The web UI for Estuary Flow.
 
 # How to install?
 
-You'll need to make sure the UI ahas the depencies installed. For the UI this should be straight forward and running `npm install` in the UI root directory.
+Make sure Node and npm are installed
 
-You'll also need internet connectivity so you can reach our Supabase API.
+Run: `npm install`
 
-# How do I run the UI?
+## I got a 401 error for `@estuary/flow-web`. 
 
-You need to run with Node 16 right now. There is an issue with Node 17.
+You need to update `~/.npmrc` with the following:
+```
+//npm.pkg.github.com/:_authToken=__YOUR_AUTH_TOKEN_YOU_MADE_ON_GITHUB__
+@estuary:registry=https://npm.pkg.github.com/
+```
 
-`npm start` starts UI
+[Github docs](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry#authenticating-with-a-personal-access-token).
 
-For more details on UI commands please look at `build.md`.
+# How to start/run?
 
-# What is the status?
+Run: `npm start`
 
-The status very much a "work in progress"... or like one level above a proof of concept.
+The UI is built to hit our own instance of Supabase. That means you will need to have that running locally as well.
 
 # Building Blocks
 
-This project was started using [Create React App](https://create-react-app.dev/) and [added TypeScript](https://create-react-app.dev/docs/adding-typescript).
-
-As we are currently using [MUI](https://mui.com/core/) as our main component library so we used [their typescript template](https://github.com/mui-org/material-ui/tree/master/examples/create-react-app-with-typescript) as a reference for this project. We combined some of their tsconfig settings with those found from [typescript-cheatsheets](https://github.com/typescript-cheatsheets/react#troubleshooting-handbook-tsconfigjson).
+-   [MUI](https://mui.com/core/) for components
+-   [Zustand](https://github.com/pmndrs/zustand) for local state
+-   [JSON Forms](https://github.com/eclipsesource/jsonforms) for many forms
+-   [SWR](https://github.com/vercel/swr) for server calls
```

Comment:
Any reason why we would not want to mention Apache ECharts here?

Reply:
That's fair... I mainly was picking ones that were really important and that one counts... I'll add.